apps/provider-console/src/context/CustomChainProvider/CustomChainProvider.tsx

```diff
@@ -24,9 +23,8 @@ export function CustomChainProvider({ children }: Props) {
       assetLists={assetLists}
       wallets={[...keplr, ...leap]}
       sessionOptions={{
-        duration: 31_556_926_000, // 1 year
```

Comment:
Any reason you removed this comment?

Reply:
I just removed all the comments, do we need that comment?

Reply:
Well comments have a use sometimes, when it's not commentted out code. For this particular instance, knowing this value means 1 year can save some time to understand without a comment in the future.

Reply:
Yeah, I agree. Going forward, if we are using an unknown number, we will add a comment on it.